sympy/polys/numberfields/modules.py

```diff
@@ -1127,6 +1161,76 @@ def __mul__(self, other):
     def _first_power(self):
         return self
 
+    def reduce_element(self, elt):
+        r"""
+        If this submodule $B$ has defining matrix $W$ in square, maximal-rank
+        Hermite normal form, then, given an element $x$ of the parent module
+        $A$, we produce an element $y \in A$ such that $x - y \in B$, and the
+        $i$th coordinate of $y$ satisfies $0 \leq y_i < w_{i,i}$.
+
+        Explanation
+        ===========
+
+        In the special case where $A$ is a power basis for a number field $K$,
+        and $B$ is a submodule representing an ideal $I$, this operation
+        represents one of a few important ways of reducing an element of $K$
+        modulo $I$ to obtain a "small" representative. See [Cohen00] Section
```

Comment:
```suggestion
        modulo $I$ to obtain a "small" representative. See [Cohen00]_ Section
```
Should create a clickable link.

Reply:
Thanks!